amplify-tools/amplify-tools-gradle-plugin/proguard-rules.pro

```diff
@@ -0,0 +1,21 @@
+# Add project specific ProGuard rules here.
```

Comment:
Is this needed?

Reply:
Doubtful, for a plugin. Since the plugin will only be deployed to a developer's machine, and not a customer's mobile phone. _If_ we succeed to integrate Proguard here, it will purely hamper our ability to debug the plugin, while providing no benefit.

TLDR we can safely remove this file.